### ISSUE TITLE
The express access shows: express deprecated req.param(name): Use req.params, req.body, or req.query 

### ISSUE THREAD

**Issue by marcellodesales** [strongloop/loopback#1047]
I just installed and upgraded the command line, generated a new sample, and the warns for the access is showing that the libraries are using a deprecated version of the calls.

```
Browse your REST API at http://0.0.0.0:3000/explorer
Web server listening at: http://0.0.0.0:3000/
express deprecated req.param(name): Use req.params, req.body, or req.query instead node_modules/loopback/common/models/access-token.js:174:16
express deprecated req.param(name): Use req.params, req.body, or req.query instead node_modules/loopback/lib/application.js:299:60
express deprecated req.param(name): Use req.params, req.body, or req.query instead node_modules/loopback/node_modules/strong-remoting/lib/http-context.js:142:18
express deprecated req.param(name): Use req.params, req.body, or req.query instead node_modules/loopback/node_modules/strong-remoting/lib/http-context.js:153:22
```

It would be great to get it fixed!

thanks!

**Comment by bajtos** [comment]
Fixed by #1038

**Comment by marcellodesales** [comment]
@bajtos perfect! Is the latest upgraded? thanks!

**Comment by bajtos** [comment]
@marcellodesales yes, the fix was released in`loopback@2.11.0`.

**Comment by marcellodesales** [comment]
@bajtos thanks! Ready for NodeSummit! :) I will say hi to the booth today!
